### DISCUSSION TITLE
1.3.0

### DISCUSSION THREAD

**Discussion by louislam** [louislam/dockge#233] [Announcements]
### 🆕 New Features
- #218 #231 Added support for editing `.env` file (Thanks @syko9000)
- #212 Added `pnpm run reset-password`, which is used for resetting your password.

### Improvements
- #228 Sort non-managed stack to the end of stack list
- #227 Improved performance

### 🐞 Bug Fixes
- #206 Fixed an issue that the container terminal could be wrong if containers from different stacks had the same service name (Thanks @ThalesC)

### 🦎 Translation Contributions
- #190 #229 Translations update from Kuma Weblate (Thanks @401U @Alanimdeo @D3-3109 @FreddieDK @LevantinLynx @MSalim7 @Zaid-maker @abosaad11 @bjornclauw @bunducdragos @cyril59310 @furkanipek @lucasreiners @nazo6 @stanol @syko9000)
- Added 日本語 (Thanks @nazo6)
- Added Dansk (Thanks @FreddieDK)
- Added Română (Thanks @bunducdragos)
- Added Nederlands (Thanks @bjornclauw)

### Others
- Other small changes, code refactoring and comment/doc updates in this repo:
@Zaid-maker
 #191 #210 #211

<hr /><em>This discussion was created from the release <a href='https://github.com/louislam/dockge/releases/tag/1.3.0'>1.3.0</a>.</em>

**Comment by gmdelisle** [comment]
Just want to say thanks.
Made installing containers with .env files like Immish a total breeze. Love this project! 🙏

**Comment by Zaid-maker** [comment]
Louis you again forget to make release for 1.3.2 😄
